lib/slash-commands/ext/ext.js

```diff
@@ -41,31 +52,96 @@ class Ext {
             callback = arguments[1];
         }
 
-        Util.readJsonFromFile(FILE, (err, data) => {
-            if (err) {
-                throw err;
+        if (this._attemptedToReadSpreadsheetConfigOnce) {//if we already tried to load the config file on a previous request...
```

Comment:
Bird's eye view of this if/else block here:
1. It is inside of a method with a name of `find`
2. The line that actually will **find** stuff is the ones with `this._handle`, all the other dozens ones has little to do with the action of *finding* itself

I suggest extracting that logic into a separate method, one only to deal with this config logic, and then call it from inside of find... It will be easier to understand what each of them do, and concerns will be more separated :)